### ISSUE TITLE
Inference on multiple gpus 

### ISSUE THREAD

**Issue by Aradhya0510** [CSAILVision/semantic-segmentation-pytorch#166]
Is it possible to perform inference/test on multiple gpus. Right now the test script seems to accepts only one gpu id and testing on large dataset very slow.

**Comment by hangzhaomit** [comment]
Yes, it should be very straight forward if you write a script similar to ```eval_multipro.py```, which is for multi-gpu evaluation.
